docs/0.1/cli_references.md

```diff
@@ -18,3 +18,413 @@ subcommand. The synopsis for each command shows its parameters and their usage.
+    -q, --quiet      Do not display output
+    -V, --version    Prints version information
+    -v               Log verbosely
+
+ARGS:
+    <path>    Path to yaml file containing a list of products
+```
+
+### grid product delete
+
+Delete an existing product.
+
+```
+Delete a product
+
+USAGE:
+    grid product delete [FLAGS] <product_id> <product_type>
+
+FLAGS:
+    -h, --help       Prints help information
+    -q, --quiet      Do not display output
+    -V, --version    Prints version information
+    -v               Log verbosely
+
+ARGS:
+    <product_id>      Unique ID for a product
+    <product_type>    Type of product (e.g. GS1
+
+```
+
+### grid product show
+
+Show details for a given product.
+
+```
+Show product specified by ID argument
+
+USAGE:
+    grid product show [FLAGS] <product_id>
+
+FLAGS:
+    -h, --help       Prints help information
+    -q, --quiet      Do not display output
+    -V, --version    Prints version information
+    -v               Log verbosely
+
+ARGS:
+    <product_id>    ID of product
+```
+
+### grid product list
+
+List all products available.
+
+```
+List currently defined products
+
+USAGE:
+    grid product list [FLAGS]
+
+FLAGS:
+    -h, --help       Prints help information
+    -q, --quiet      Do not display output
+    -V, --version    Prints version information
+    -v               Log verbosely
+```
+
+### grid schema create
+
+Create a schema definition via the Schema smart contract.
+
+```
+Create schemas from a yaml file
+
+USAGE:
+    grid schema create [FLAGS] <path>
+
+FLAGS:
+    -h, --help       Prints help information
+    -q, --quiet      Do not display output
+    -V, --version    Prints version information
+    -v               Log verbosely
+
+ARGS:
+    <path>    Path to yaml file containing a list of schema definitions
+```
+
+### grid schema update
+
+Update an existing schema definition via the Schema smart contract.
+
+```
+Update schemas from a yaml file
+
+USAGE:
+    grid schema update [FLAGS] <path>
+
+FLAGS:
+    -h, --help       Prints help information
+    -q, --quiet      Do not display output
+    -V, --version    Prints version information
+    -v               Log verbosely
+
+ARGS:
+    <path>    Path to yaml file containing a list of schema definitions
+```
+
+### grid schema show
+
+Show details for a specific schema.
+
+```
+Show schema specified by name argument
+
+USAGE:
+    grid schema show [FLAGS] <name>
+
+FLAGS:
+    -h, --help       Prints help information
+    -q, --quiet      Do not display output
+    -V, --version    Prints version information
+    -v               Log verbosely
+
+ARGS:
+    <name>    Name of schema
+```
+
+
+### grid schema list
+
+List all available schemas.
+
+```
+List currently defined schemas
+
+USAGE:
+    grid schema list [FLAGS]
+
+FLAGS:
+    -h, --help       Prints help information
+    -q, --quiet      Do not display output
+    -V, --version    Prints version information
+    -v               Log verbosely
+```
```

Comment:
Missing the output for `gridd` as defined in https://github.com/hyperledger/grid/blob/master/daemon/src/main.rs

Reply:
Would that actually belong in the CLI reference? `gridd` is not a part of the CLI it's the separate executable to start the daemon.

Reply:
If we're using Splinter as a model, the Splinter CLI reference includes `splinterd` and `scabbard`.  @vaporos already approved the PR without this information, so I'm okay with not including `gridd` now, and making a note to add it later.